dhis2/deploy.sh

```
@@ -0,0 +1,2 @@
+sudo cp dhis-integration-1.0-SNAPSHOT.jar /opt/dhis-integration/bin/dhis-integration.jar
```

Shellcheck (0.10.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)